pkgs/development/python-modules/kiss-headers/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, buildPythonPackage, fetchFromGitHub, python, requests, pytestCheckHook, pytest-cov }:
+
+buildPythonPackage rec {
+  pname = "kiss-headers";
+  version = "2.3.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "Ousret";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-/eTRyxFyAKQMzE/JjdoEN3w0lRiaIJcsJHTWV8M0CYQ=";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-cov
```

Comment:
```suggestion
```
Please remote the requirement on cov.

Reply:
Sorry, didn't get what to do here. If I remove pytest-cov, the checks will throw the following error:

```
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --cov=kiss_headers --cov-report=term-missing
  inifile: /build/source/setup.cfg
  rootdir: /build/source
```

Reply:
You want the following:

```nix
postPatch = ''
  substituteInPlace setup.cg \
    --replace "--cov=kiss_headers --doctest-modules --cov-report=term-missing -rxXs" "--doctest-modules -rxXs"
'';
```